docs/content/en/index.md

```diff
@@ -19,3 +19,8 @@ Now it's easy to create tests for projects and modules using Nuxt.
 ## Key features
 
 <list :items="items"></list>
+
+## Nuxt Framework vs Vue Component Testing
+@nuxt/test-utils will build an app (or generate a website), launch a server and will provide a browser (all are optional). For example, @nuxt/test-utils might help to check if Nuxt routes, middleware, modules, etc work and if pages are rendered as expected (e.g. collection of components). Focus is on routes, browsing and Nuxt instance, but not Vue components.
+
+Use [@vue/test-utils](https://vue-test-utils.vuejs.org/) to test single Vue components. (vue/test-utils does not build Nuxt apps and knows nothing about Nuxt instances)
```

Comment:
```suggestion
## Why `@nuxt/test-utils`?

This library allows you to test your Nuxt app or module. It includes unit and end-to-end test helpers, but is not intended as a substitute for component test libraries like `@vue/test-utils`.

With this library you can interact with a Nuxt instance - whether that is verifying a plugin or module is behaving as expected, getting a fully-rendered server response or conducting client-side browser tests using playwright.
```

Reply:
Thanks, this makes things much clearer for newcomers, especially adding "e2e" while now mentioning @vue/test-utils for component testing.

Just one question: You write "it includes unit ... test helpers". Can you specify what unit helpers are, in context of nuxt-test-utils?
For example, atm I use ts-jest to test my functions that are separated into files. How would this package help me do unit testing?